src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/TreeController.cs

```diff
@@ -81,7 +79,7 @@ public async Task<IActionResult> List(AdminTreeListOptions options, PagerParamet
 
             var startIndex = pager.GetStartIndex();
             var pageSize = pager.PageSize;
-            IEnumerable<AdminTree> results = new List<AdminTree>();
+            IEnumerable<Models.AdminMenu> results = new List<Models.AdminMenu>();
 
             //todo: handle the case where there is a deserialization exception on some of the presets.
             // load at least the ones without error. Provide a way to delete the ones on error.
```

Comment:
"Error when retrieving the list of admin menus"

Reply:
You means with an Admin Menu that would come from the old module?

Reply:
No. Sorry. Not used to do reviews on github. :)
A few lines below there is an error message that needs to be changed from "Error when retrieving the list of admin trees" to "Error when retrieving the list of admin menus"

Reply:
ok I fixed it.